geth/account/accounts.go

```diff
@@ -31,7 +32,9 @@ type GethServiceProvider interface {
 
 // Manager represents account manager interface.
 type Manager struct {
-	geth            GethServiceProvider
+	geth GethServiceProvider
+
```

Comment:
Why this extra line? And I typically know mutex with it's more infrastructure character opposite to the value fields of a struct as the first field:

```
type Manager struct {
    mu   sync.RWMutex
    geth GethServiceProvider
    ...
}
```

Reply:
To indicate that `mu` guards `selectedAccount`. But it can guard `geth` if that would be assignable.